prometheus/collectors/go_collector_go120_test.go

```diff
@@ -33,6 +30,9 @@ func withAllMetrics() []string {
 		"go_cpu_classes_scavenge_total_cpu_seconds_total",
 		"go_cpu_classes_total_cpu_seconds_total",
 		"go_cpu_classes_user_cpu_seconds_total",
+		"go_gc_cycles_automatic_gc_cycles_total",
+		"go_gc_cycles_forced_gc_cycles_total",
+		"go_gc_cycles_total_gc_cycles_total",
```

Comment:
After running the script, the order changed 😬

Reply:
The metrics were not sorted in the previous files

https://github.com/prometheus/client_golang/pull/1483#discussion_r1545534373